ONSdigital/census-rm-case-processor
# Handle refusal event

# Motivation and Context
Case processor is required to handle a refusal


# What has changed
Case records refusal status.
Refactored event logging in to separate class.
Created new case.refusals queue config.
Correct receipt incoming message as per event spec.





# How to test?
check this and related branches below, build docker images and run with docker-dev.
run tests




# Links
[Trello](https://trello.com/c/29ZsK7ci/825-rmc-197-handle-refusal-received-event-rev13-8)




# Screenshots (if appropriate):
